app/controllers/api/motions_controller.rb

```diff
@@ -25,8 +25,11 @@ def update_outcome
   end
 
   def index
-    load_and_authorize :discussion
-    instantiate_collection { |collection| collection.where(discussion: @discussion).order(:created_at) }
+    load_and_authorize :discussion if params[:discussion_id] || params[:discussion_key]
+    instantiate_collection do |collection|
+      collection = collection.where(discussion: @discussion) if @discussion
```

Comment:
This'll be a common pattern, I reckon, in the index action... either 'give me all the discussions for this group' or 'all the groups for this user' or 'all the motions for this discussion' or whatever... I reckon pulling that behaviour out somehow might be nice.